object-storage-core/src/main/java/io/micronaut/objectstorage/DefaultInputStreamMapper.java

```diff
@@ -29,7 +30,7 @@
 public class DefaultInputStreamMapper implements InputStreamMapper {
 
     @Override
-    public byte[] toByteArray(InputStream inputStream) {
+    public byte[] toByteArray(@NonNull InputStream inputStream) throws ObjectStorageException {
```

Comment:
Palo designed `ObjectStorageException` to be unchecked. It could be debatable whether we should make it checked, and then add `throws` clauses through the entire API (now it's only in a few places).

In any case, this change is unrelated to the PR.

Reply:
I'd leave it unchecked and omit from throws clauses, and maybe add a note in the docs that users can add try/catch if they want

Reply:
I added it the throws clause and the javadoc to document the API.

Reply:
I have removed the `throws` clause from this PR. 

Please note that we have ` throws ObjectStorageException` in other API such as `ObjectStorageOperations`

https://github.com/micronaut-projects/micronaut-object-storage/blob/2673dbb250c8d78025e69a627d0cb20f27b947fa/object-storage-core/src/main/java/io/micronaut/objectstorage/ObjectStorageOperations.java#L26

Reply:
I will make it consistent in #88